apps/zero-api/src/interceptors/NoDataInterceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor, NotFoundException } from '@nestjs/common';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import * as _ from "lodash";
```

Comment:
Please do not use global imports

Reply:
importing it as `import isNil from ‘lodash/fp/isNil’` like you suggested causes `isNil_1.default is not a function` errors